### ISSUE TITLE
brakeman breaks when trying to call private methods

### ISSUE THREAD

**Issue by taf2** [presidentbeef/brakeman#846]
```bash
/Users/taf2/.rvm/gems/ruby-2.1.5@ctm/gems/brakeman-3.1.2/lib/brakeman/util.rb:445:in `truncate_table': private method `terminal_size' called for #<HighLine:0x007fce5e07a010> (NoMethodError)
	from /Users/taf2/.rvm/gems/ruby-2.1.5@ctm/gems/brakeman-3.1.2/lib/brakeman/report/report_table.rb:7:in `generate_report'
	from /Users/taf2/.rvm/gems/ruby-2.1.5@ctm/gems/brakeman-3.1.2/lib/brakeman/report.rb:73:in `generate'
	from /Users/taf2/.rvm/gems/ruby-2.1.5@ctm/gems/brakeman-3.1.2/lib/brakeman/report.rb:64:in `to_s'
	from /Users/taf2/.rvm/gems/ruby-2.1.5@ctm/gems/brakeman-3.1.2/lib/brakeman/report.rb:35:in `format'
	from /Users/taf2/.rvm/gems/ruby-2.1.5@ctm/gems/brakeman-3.1.2/lib/brakeman.rb:368:in `block in write_report_to_formats'
	from /Users/taf2/.rvm/gems/ruby-2.1.5@ctm/gems/brakeman-3.1.2/lib/brakeman.rb:367:in `each'
	from /Users/taf2/.rvm/gems/ruby-2.1.5@ctm/gems/brakeman-3.1.2/lib/brakeman.rb:367:in `write_report_to_formats'
	from /Users/taf2/.rvm/gems/ruby-2.1.5@ctm/gems/brakeman-3.1.2/lib/brakeman.rb:350:in `scan'
	from /Users/taf2/.rvm/gems/ruby-2.1.5@ctm/gems/brakeman-3.1.2/lib/brakeman.rb:62:in `run'
	from /Users/taf2/.rvm/gems/ruby-2.1.5@ctm/gems/brakeman-3.1.2/bin/brakeman:78:in `<top (required)>'
	from /Users/taf2/.rvm/gems/ruby-2.1.5@ctm/bin/brakeman:23:in `load'
	from /Users/taf2/.rvm/gems/ruby-2.1.5@ctm/bin/brakeman:23:in `<main>'
	from /Users/taf2/.rvm/gems/ruby-2.1.5@ctm/bin/ruby_executable_hooks:15:in `eval'
	from /Users/taf2/.rvm/gems/ruby-2.1.5@ctm/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by taf2** [comment]
fixed by upgrading highline

Installing highline 1.7.8 (was 1.6.16)

**Comment by presidentbeef** [comment]
FWIW this dependency issue was fixed in Brakeman 3.1.3: http://brakemanscanner.org/blog/2015/12/02/brakeman-3-dot-1-3-released/
